test/e2e/docs/style-guide-playwright.md

```diff
@@ -263,41 +277,125 @@ async function openModal() {
 
 ## Selectors
 
-### Type
+Selectors form the core of any automated e2e test scripts. For an overview please refer to the [MDN page on CSS selectors](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors).
+
+Ideally, a selector satisfies all of the following:
+
+- **unique**: one selector, one element.
+- **reliable**: the same element is selected with each iteration.
+- **brief**: selector is short and easy to read.
+
+Place selectors within the same file, but outside of the class representing the object. Never place a selector within the class definition itself.
+
+**Avoid**:
+
+```typescript
+class SomeObject() {
+	submitButtonSelector: '#submit'
+}
+```
+
+**Instead**:
+
+```typescript
+const selectors = {
+	submitButton: '#submit',
+}
+
+class SomeObject() {
+	// class things
+}
+
+```
+
+Selectors should be defined in an key:value object mapping as follows:
+
+```typescript
+const selectors = {
+	// Buttons
+	submitButton: '#submit',
+	cancelButton: '#cancel',
+}
+```
```

Comment:
Do we want to add a blurb here for the idea of "Dynamic" selectors? I.e., selectors that are functions and take a parameter that they inject into the selector string?

Reply:
I think it will be a good idea to demonstrate it here, yeah.